net-misc/ubridge/ubridge-0.9.13.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils fcaps user
+
+DESCRIPTION="Bridge for UDP tunnels, Ethernet, TAP and VMnet interfaces."
+HOMEPAGE="https://github.com/GNS3/ubridge"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+system-iniparser"
+
+RDEPEND="
+	net-libs/libpcap
+	system-iniparser? ( dev-libs/iniparser )
+"
+
+pkg_setup() {
+	enewgroup ubridge
+}
+
+src_prepare() {
+	epatch "${FILESDIR}"/${P}-respect-flags.patch
```

Comment:
we dont want epatch and eutils in EAPI 6 anymore. Put the patch in global `PATCHES` array and nix `src_prepare`